server/protocols/jmap/src/test/java/org/apache/james/jmap/DefaultMailboxesProvisioningFilterThreadTest.java

```diff
@@ -84,7 +89,34 @@ public void testConcurrentAccessToFilterShouldNotThrow() {
         AnnotatedTestRunner runner = new AnnotatedTestRunner();
         runner.runTests(this.getClass(), DefaultMailboxesProvisioningFilter.class);
     }
-    
+
+    private static class FakeSubscriptionManager implements SubscriptionManager {
```

Comment:
please use a mock instead

Reply:
You can not! It's due to that concurrent testing framework... Sorry, it do not combine well

Reply:
ok